### DISCUSSION TITLE
Update 20240219 issue

### DISCUSSION THREAD

**Discussion by Roland-B** [rern/rAudio#1606] [Bug Reports]
Latest update completely broke raudio1, my device is blank and no way to change anything. Was working fine with all prev updates until this one.

Updated showed:

Invalid card number 'scontents'.
/srv/http/bash/settings/player-conf.sh: line 29:  8462 Killed                  $dirbash/status-push.sh
/srv/http/bash/common.sh: line 171: echo: write error: Broken pipe

rAudio 20240219
6.6.16-3-rpi aarch64
Raspberry Pi 4B Rev 1.2
BCM2711 • 3.7 GiB
4 x Cortex-A72 @ 1.50 GHz

**Comment by ceejayemm** [comment]
Have a look at #1594 and #1600, #1602 & #1603. It's being worked but not fully fixed just yet. 

Chris

**Comment by Roland-B** [comment]
@ceejayemm Thanks but in my case I have only had issue with my USB DAC only on this v latest update.. so this update to fix other people's issues has caused mine to break it seems!? So thought would require separate issue raised.

**Comment by Roland-B** [comment]
@rern managed to get device back by turning off volume control and back on and then re-enabling no volume. The only issues I have discovered thus far are:  

1. When play via spotify get no cover and/or just a flashing raudio logo, and occasionally a spotify api renewal failure flashing up but it still plays spotify content. 

2. Device is listed as USB Audio instead of tha actual USB DAC model name.

Seems a bit of a muddle at the moment after few months of everything nicely, hope not too much work for you to get it all working nicely, and best of luck. 

Thanks,
Roland.

**Reply by Roland-B** [reply]
Is almost there fixing the issues after last 2 updates. DAC now shows using proper name for the first time.

@rern Only issue now is now is "no volume" is not working properly. It gets set to 50% for no discernable reason. Have to SSH in and run alsamixer to set it to 100%!? (Volume Type is set to DAC Hardware). When reboot it gets set back to 50%!?


Roland

rAudio 202403031
6.6.18-1-rpi aarch64
Raspberry Pi 4B Rev 1.2
BCM2711 • 3.7 GiB
4 x Cortex-A72 @ 1.50 GH

**Reply by Roland-B** [reply]
Fixed in update 20240312.
